components/TopBar.js

```diff
@@ -303,11 +309,16 @@ const TopBar = ({ showSearch, menuItems, showProfileAndChangelogMenu }) => {
                     <FormattedMessage defaultMessage="Explore" />
                   </MainNavItem>
                 </Link>
-                <Link href="/help">
-                  <MainNavItem as={Container} isActive={isRouteActive('/help')}>
-                    <FormattedMessage defaultMessage="Help & Support" />
+                <StyledTooltip
```

Comment:
This is just my opinion, but:
I think it would be simpler, more consistent, and more harmonic to write down "Search" on the button label than having this tooltip.

Reply:
Good point, as this button behaves differently from the other nav items I wanted some distinction for it, took the opportunity to refactor it a bit, removing the tooltip and spelling out "Search..." while giving it a slightly different look and also including a keyboard shortcut:

![image](https://github.com/opencollective/opencollective-frontend/assets/1552194/9e33164d-e9bf-488f-bcaf-4b0b0964da28)